<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorChromaWidget</class>
 <widget class="QWidget" name="InspectorChromaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutKey">
     <item>
      <widget class="QLabel" name="labelKey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>75</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;Key&quot; style=&quot;color: #BEBEBE;&quot;&gt;Key&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>comboBoxKey</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxKey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowMask">
       <property name="text">
        <string>Show mask</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutThreshold">
     <item>
      <widget class="QLabel" name="labelThreshold">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;Threshold&quot; style=&quot;color: #BEBEBE;&quot;&gt;Threshold&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sliderThreshold</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderThreshold">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxThreshold">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSoftness">
     <item>
      <widget class="QLabel" name="labelSoftness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;Spread&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#bebebe;&quot;&gt;Softness&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sliderSoftness</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderSoftness">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSoftness">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBlur">
     <item>
      <widget class="QLabel" name="labelBlur">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;Chroma Keying&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#bebebe;&quot;&gt;Blur Mask&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sliderBlur</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderBlur">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBlur">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSpill">
     <item>
      <widget class="QLabel" name="labelSpill">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;Spill&quot; style=&quot;color: #BEBEBE;&quot;&gt;Spill&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sliderChromaSpill</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderChromaSpill">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSpill">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerChroma">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxKey</tabstop>
  <tabstop>sliderThreshold</tabstop>
  <tabstop>spinBoxThreshold</tabstop>
  <tabstop>sliderSoftness</tabstop>
  <tabstop>spinBoxSoftness</tabstop>
  <tabstop>sliderChromaSpill</tabstop>
  <tabstop>spinBoxSpill</tabstop>
  <tabstop>sliderBlur</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSoftness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderSoftness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderBlur</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSpill</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderChromaSpill</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderThreshold</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBoxThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSoftness</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBoxSoftness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlur</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBoxBlur</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderChromaSpill</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBoxSpill</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSoftness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setSoftness(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setBlur(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSpill</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setSpill(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxKey</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setKey(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowMask</sender>
   <signal>toggled(bool)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>setShowMask(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>641</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelSpill</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>resetSpill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>-13</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelBlur</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>resetBlur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>-10</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelSoftness</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>resetSoftness()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>-13</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelThreshold</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>resetThreshold()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>-14</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelKey</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorChromaWidget</receiver>
   <slot>resetKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>-17</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setThreshold(int)</slot>
  <slot>setSoftness(int)</slot>
  <slot>setBlur(int)</slot>
  <slot>setSpill(int)</slot>
  <slot>setKey(QString)</slot>
  <slot>setShowMask(bool)</slot>
  <slot>resetSpill()</slot>
  <slot>resetBlur()</slot>
  <slot>resetSoftness()</slot>
  <slot>resetThreshold()</slot>
  <slot>resetKey()</slot>
 </slots>
</ui>
